changelogs/unreleased/improved-agent-typing.yml

```diff
@@ -0,0 +1,6 @@
+description: Add a return after an error log
+change-type: patch
+destination-branches:
+  - iso4
+  - iso5
+  - master
```

Comment:
```suggestion
```

Reply:
I should have indeed thought of that. Thanks!